In [1]:
import numpy as np
import pandas as pd
from scipy import stats
import seaborn as sns
import matplotlib.pyplot as plt

EXERCISE 1.

Answer the following questions by looking at the distribution table or coding with Python.
1.	A salesperson has found that the probability of a sale on a single contact is approximately .3. If the salesperson contacts 10 prospects, what is the approximate probability of making at least one sale? (Wackerly, Ex.3.135)


In [16]:
n = 10
p = 0.03 
x = 0 

In [17]:
1-stats.binom.pmf(x, n, p)

0.2625758731050719

2.	Ten coins are tossed simultaneously. Find the probability of getting

(i) at least seven heads


In [20]:
n = 10
p = 0.5 
x = 6

In [22]:
1-stats.binom.cdf(x, n, p)

0.171875

(ii) exactly seven heads

In [23]:
x=7

In [25]:
stats.binom.pmf(x, n, p)

0.11718749999999999

(iii)at most seven heads

In [26]:
stats.binom.cdf(x, n, p)

0.9453125

EXERCISE 2.

The cycle time for trucks hauling concrete to a highway construction site is uniformly distributed over the interval 50 to 70 minutes. What is the probability that the cycle time exceeds 65 minutes  if it is known that the cycle time exceeds 55 minutes? (Wackerly, Ex.4.51)


In [30]:
a = 50
b = 70
dist = stats.uniform(a,b-a)
c=1-Dist.cdf(65)
c

0.25

EXERCISE 3. 

The width of bolts of fabric is normally distributed with mean 950 mm (millimeters) and standard deviation 10 mm.
1.	What is the probability that a randomly chosen bolt has a width of between 947 and 958mm?


In [37]:
mu=950
sigma= 10

stats.norm.cdf(958, mu, sigma) - stats.norm.cdf(946, mu, sigma)

0.44356634302692755

2.	What is the appropriate value for C such that a randomly chosen bolt has a width less than C with 
probability .8531? (Wackerly, Ex.4.73)

In [38]:
stats.norm.ppf(0.8531, mu, sigma)

960.4982190962642

EXERCISE 4. 

Suppose scores on exams in statistics are normally distributed with an unknown population mean and a 
population standard deviation of 3 points. A random sample of 36 scores is taken and gives a sample mean 
(sample mean score) of 68. Find a confidence interval estimate for the population mean exam score 
(the mean score on all exams). Find a 90% confidence interval for the true (population) mean of statistics 
exam scores.


In [42]:
sigma = 3
n = 36
x = 68

In [44]:
stats.norm.interval(alpha=0.90, loc=x,scale= sigma/np.sqrt(n))

(66.35514637304853, 69.64485362695147)

EXERCISE 5.  

What is the normal body temperature for healthy humans? A random sample of 130 healthy human body 
temperatures provided by Allen Shoemaker yielded 98.25 degrees and standard deviation 0.73 degrees.
1.	Give a 99% confidence interval for the average body temperature of healthy people.


In [50]:
sigma = 0.73
n = 130
x = 98.25
stats.norm.interval(alpha=0.99, loc=x,scale= sigma/np.sqrt(n))

(98.08508192246582, 98.41491807753418)

2.	Does the confidence interval obtained in part (a) contain the value 98.6 degrees, the accepted 
average temperature cited by physicians and others? What conclusions can you draw? (Wackerly, Ex.8.60)

No. 98.6 degrees not within the 99% confidence interval.
but 98.41 degrees is close to the final limit of the confidence interval.

GÜVEN ARALIĞI BULMA 

#from scipy import stats
#stats.norm.interval(.95, loc = 175, scale = 20/np.sqrt(40))

interval(alpha, loc=0, scale=1)

Endpoints of the range that contains fraction alpha [0, 1] of the distribution

But a comment in this post states that the actual correct way of obtaining the confidence interval is:

conf_int = stats.norm.interval(0.68, loc=mean, 
    scale=sigma / np.sqrt(len(a)))
    
    that is, sigma is divided by the square-root of the sample size: np.sqrt(len(a)).